dev-haskell/binary/binary-0.8.7.0.ebuild

```diff
@@ -18,7 +18,7 @@ SRC_URI="https://hackage.haskell.org/package/${P}/${P}.tar.gz"
 LICENSE="BSD"
 SLOT="0/${PV}"
 # keep in sync with ghc-8.8
```

Comment:
We need version 0.8.8.0 to be compatible with ghc-9.0.2. All in `::haskell` ready to go, so you should be able to pull it over to here with only minor changes if any.